config/sync/user.role.health_condition_supervisor_user.yml

```diff
@@ -36,6 +36,24 @@ permissions:
   - 'moderate content from needs_review to draft'
   - 'moderate content from needs_review to published'
   - 'moderate content from published to needs_review'
+  - 'revert application revisions'
```

Comment:
Do Health condition supervisors need edit permissions for all content types?

Reply:
Nope!